protocols/raft/src/main/java/io/atomix/protocols/raft/roles/FollowerRole.java

```diff
@@ -241,9 +254,9 @@ protected VoteResponse handleVote(VoteRequest request) {
    * Cancels the heartbeat timer.
    */
   private void cancelHeartbeatTimers() {
```

Comment:
Is this used? If so, we can use it above on line 221; otherwise we can just remove this method

Reply:
It is used in the method below: 

```
  @Override	  @Override
  public synchronized CompletableFuture<Void> stop() {	  public synchronized CompletableFuture<Void> stop() {
    raft.getMembershipService().removeListener(clusterListener);	    raft.getMembershipService().removeListener(clusterListener);
    return super.stop().thenRun(this::cancelHeartbeatTimers);	    return super.stop().thenRun(this::cancelHeartbeatTimers);
  }
```